CS342MemberViews/Sources/ShriyaReddy/HobbyButton.swift

```diff
@@ -0,0 +1,49 @@
+//
+// File.swift
+// Created by Shriya Reddy on 1/24/23.
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+private var description: String {
+        guard let descriptionPath = Bundle.module.path(forResource: "HobbyText", ofType: "txt"),
+              let description = try? String(contentsOfFile: descriptionPath) else {
+            return ""
+        }
+        
+        return description
+    }
```

Comment:
Please add properties (including this computed properties) into the scope of a type that they are used to, e.g., the `HobbyButton` view. You can simply move it into the struct by adding it above or below the `body` computed propert.

You have correctly added the description property to the `IntroButton` in the right way.